src/api/tests/service/creator_analytics/test_query.py

```
@@ -560,6 +561,168 @@ def test_user_users_lookup_without_where_user_bid_is_rejected(
+            "table": "user_users",
+            "select": ["user_bid", "user_identify"],
+            "where": [{"field": "user_identify", "op": "=", "value": "13800138000"}],
+            "limit": 1,
+        },
+    )
+
+    payload = response.get_json(force=True)
+    assert payload["code"] == 0
+    rows = payload["data"]["rows"]
+    assert len(rows) == 1
+    row = dict(zip(payload["data"]["columns"], rows[0]))
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)